### PR TITLE
Markdown formatting  in csv table

### LINKED ISSUE #767: Append information on markdown formatting in CSV tables to contributor guide
Please try to avoid the use of heavily-formatted CSV tables as much as possible. In some cases they are necessary, but require some careful formatting because the markdown linter does not work within CSV files.

- To force text onto a new line use `<br>` with no surrounding whitespace.
- To force text onto a new line, and indent the new line, use `<br>&nbsp;  `.
    - Note the two trailing spaces!
    - Use two spaces for consistency with unordered lists.
    - Avoid usage of ordered lists and nested lists.
    - If you need intricate formatting, the content may belong outside the table, with a link to the content in the table.
- To create an anchor for use with internal links, surround the leading part of the text with `<span id='$anchor-name'>text</span>`. Replace `$anchor-name` with the actual anchor you wish to use in the internal link and HTTPS URL.
    - Only use lowercase letters, numerals, and the dash `-` character. 
    - The dash `-` character should only be used to separate words, or where a dash or hyphen would normally be placed.
    - Do not use any other punctuation or characters.
    - Make the anchor match the surrounded text.
    - Only surround a small amount of text that "feels" like a header.
    - Example: "Lab PI's" becomes `lab-pis`.
    - This maintains consistency with section anchors produced by the MkDocs Material theme.

### PR DESCRIPTION
**Pull request by bdu-birhanu**
# Pull Request

## Proposed Changes
 - Added information on markdown formatting in csv tables

## Related Issues
Fixes #767

### CODE REVIEW

File: docs/contributing/contributor_guide.md [hunk 1]
```diff
@@ -96,7 +96,21 @@ Sometimes the Table of Contents, rendered on the right-hand side of pages, can g
...
 Prefer to use commas `,` as separators, quoting entries with double quotes `"` where necessary. These are required by the default table reader plugin settings. Formatting and style are not automated at this time.
 
-Most simple markdown formatting used within CSV Files will be rendered as expected. We encourage cross-linking from within tables to other parts of the documentation, or linking to external resources, as appropriate. Most VSCode markdown extension features do not function within CSV tables, and these are not linted at this time.
+Most simple markdown formatting used within CSV Files will be rendered as expected, while other complex markdown features might not display as expected. To enhance readability:
+
+- Avoid complex formatting and limit heavy styling inside CSV tables, as markdown linters often bypass content in these tables.
+- To control line breaks within table cells:
+    - Use `<br>` without surrounding spaces for a new line.
+    - Use `<br>&nbsp;` to create a new, indented line within the cell. Including the extra two spaces enhances consistency with unordered lists elsewhere in markdown.
+
```
Comment (wwarriner) [inline docs/contributing/contributor_guide.md:105]: These blank lines between unordered list items will be removed when rendered. Let's remove them also in the source document.

File: docs/contributing/contributor_guide.md [hunk 2]
```diff
@@ -96,7 +96,21 @@ Sometimes the Table of Contents, rendered on the right-hand side of pages, can g
...
+- CSV tables don’t support ordered or nested lists effectively. As a best practice:
+    - Stick to simple bullet lists with two-space indentation.
+    - If a list requires multiple levels or detailed formatting, consider placing the content outside the table. A link to this content can be added to the table to maintain context.
+
```
Comment (wwarriner) [inline docs/contributing/contributor_guide.md:109]: Same as above.

File: docs/contributing/contributor_guide.md (hunk 1 above)
Comment (wwarriner) [reply docs/contributing/contributor_guide.md:105]: Alternatively, provide higher-level list items to group the existing list items.

**Review by wwarriner — CHANGES_REQUESTED** [review]
Great, thank you! Only one request to deal with some blank lines in the source document.

File: docs/contributing/contributor_guide.md (hunk 1 above)
Comment (bdu-birhanu) [reply docs/contributing/contributor_guide.md]: Fixed, Thank you.

**Review by wwarriner — APPROVED** [review]

**Pull request merged**